### PR TITLE
playwright: init at 1.13.1

### PR DESCRIPTION
**Pull request by jlesquembre**
###### Motivation for this change
WIP
For now, only chromium is on the derivation, but I'd like to add firefox and webkit.
I'm not sure if that's is the right approach to create a playwright package, opening this PR to get feedback

I took some inspiration from https://github.com/ludios/nixos-playwright

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/web/playwright/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  chromium-rev = "901522";
+  firefox-rev = "1278";
+in
+
+stdenv.mkDerivation rec {
+  version = "1.13.1";
+  pname = "playwright";
+
+  src = fetchurl {
+    url = "https://playwright.azureedge.net/builds/chromium/${chromium-rev}/chromium-linux.zip";
```
Comment (SuperSandro2000) [inline pkgs/development/web/playwright/default.nix:22]: Is this a chromium binary? If so we should use the one from nixpkgs.

Comment (jlesquembre) [reply pkgs/development/web/playwright/default.nix:22]: It is. The main issue to create a playwright package is that playwright compiles the 3 browsers (chromium, firefox and webkit) with some patches. My idea here is to get the generated binaries and do what we are doing for google-chrome and firefox-bin. Ideally, I'd like to reuse what we are doing on those packages, that's why this derivation depends on google-chrome at build time

**Comment by bbigras** [comment]
Any progress on this?

**Comment by jlesquembre** [comment]
@bbigras no, I don't have plans to work on it. If anyone wants to continue working on it, feel free to do it, I hope this PR can be used as a starting point

**Comment by SuperSandro2000** [comment]
Closing in favor of #184511

**Pull request closed without merging**